packages/ts/frontend/src/CsrfInfoSource.ts

```diff
@@ -1,6 +1,8 @@
 /// <reference lib="webworker" />
 // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
-const cookieManagerPromise = globalThis.document ? import('./CookieManager.js') : undefined;
+import { VAADIN_BROWSER_ENVIRONMENT } from './utils.js';
+
+const cookieManagerPromise = VAADIN_BROWSER_ENVIRONMENT ? import('./CookieManager.js') : undefined;
```

Comment:
It seems that this conditional import isn't needed here(?), since the only usage in `extractCsrfInfoFromMeta` is always given a non-null document as an argument.

Reply:
Probably the function is never called from service workers, but still the import needs to be done conditionally to avoid an error.

Reply:
We've tried a different technique to make Vite drop the Cookie Manager code from the service worker, and turns out declaring `"sideEffects"` is even better.